### ISSUE TITLE
[New Feature] Support Web Dashboard for User-friendly Watching.

### ISSUE THREAD

**Issue by jacklanda** [vllm-project/vllm#2662]
Hey there, vLLM is a terrific project for me.

There is a natural question: 
Is there any plan to support the **web dashboard** that could be deployed simultaneously as the inference engine of vLLM?
I mean this could help users observe, diagnose, and manage the inference system easily, although I am a faithful CLI user :)

Any thoughts about this?

**Comment by jacklanda** [comment]
Any thoughts about this?

**Comment by hmellor** [comment]
No, vLLM is an inference engine that provides an API for querying LLMs.

Since vLLM provides an OpenAI compatible API, you can use any frontend that is compatible with the OpenAI API, such as https://github.com/lm-sys/FastChat

**Comment by jacklanda** [comment]
> No, vLLM is an inference engine that provides an API for querying LLMs.
> 
> Since vLLM provides an OpenAI compatible API, you can use any frontend that is compatible with the OpenAI API, such as https://github.com/lm-sys/FastChat

Okay, thanks for replying.

**Comment by hmellor** [comment]
Rereading your original post, I think I may have misunderstood what you were asking for.

Now that vLLM has Prometheus metrics, would a grafana dashboard match the description of what you're asking for? You can use Loki as well so that any logged text can also be processed and viewed in the grafana dashboard.

**Comment by hmellor** [comment]
See https://github.com/vllm-project/vllm/tree/main/examples%2Fproduction_monitoring for an example implementation
